docs/src/main/tut/client.md

```diff
@@ -50,7 +50,7 @@ speaks HTTP 1.x.
 ```tut:book
 import org.http4s.client.blaze._
 
-val httpClient = PooledHttp1Client[IO]()
+val httpClient = Http1Client[IO]().unsafeRunSync
```

Comment:
I get what's going on here, but this runs against "unsafeRunSync only at the end of the world."  But putting the entire tut into a comprehension also won't work.  Maybe we need to explain ourselves here?

Reply:
This is a hard issue as best practice would be using the streaming iterface and using it only where we need it, probably in multiple blocks calling different clients. Although in an app you generally want 1 client bracketed near the beginning.

Reply:
Yeah, I think in some of the other ones we can flatmap our way to victory.  This one, since it's a longer narrative, is the chewy one.

Reply:
At a minimum we need to add comments on why this was done.